src/test/java/com/wikia/webdriver/testcases/adstests/TestTopWamWikis.java

```diff
@@ -11,7 +11,7 @@
 public class TestTopWamWikis extends TemplateNoFirstLoad {
 
   Credentials credentials = Configuration.getCredentials();
-  Integer numberOfTop1kWikis = 997;
+  Integer numberOfTop1kWikis = 999;
```

Comment:
I would say → no. :wink: 
(feel free to change it for checking number in range, e.g. between 900 and 1100)